src/hooks/useBalanceChartData/cosmosUtils.ts

```diff
@@ -1,32 +1,5 @@
-import { ASSET_REFERENCE, cosmosChainId, toAssetId } from '@shapeshiftoss/caip'
-import { bnOrZero } from 'lib/bignumber/bignumber'
 import { Tx } from 'state/slices/txHistorySlice/txHistorySlice'
 
-import { Bucket } from './useBalanceChartData'
-
-const cosmosAssetId = toAssetId({
-  chainId: cosmosChainId,
-  assetNamespace: 'slip44',
-  assetReference: ASSET_REFERENCE.Cosmos,
-})
-
 export const includeTransaction = (tx: Tx): boolean =>
```

Comment:
we still need this, and i suspect we're missing a method here causing residual balances for cosmossdk based assets

Reply:
@gomesalexandre any ideas?

Reply:
Let's sync on this one with @elmutt, this line gives a bit of context already:
https://github.com/shapeshift/web/blob/f6eee96660e6f875186779f15ca370dfa17dd996/src/hooks/useBalanceChartData/useBalanceChartData.ts#L248

tl;dr from what I remember reviewing this, and reading the PR https://github.com/shapeshift/web/pull/1354 again is un/delegate Txs have a value, and we don't want to account for the value itself on the balance charts, only the balance change